dvc.yaml

```diff
@@ -3,33 +3,29 @@
 
 # The complete process can be executed using:
 # dvc repro
-# If you want to force redoing the process use 
+# If you want to force redoing the process use
 # dvc repro -f
-# Individual stage can be executed using: 
+# Individual stage can be executed using:
 # dvc repro <stage>
 
 stages:
-  download:
-    cmd: stages/01_download.sh
+  create:
+    cmd: python3 stages/01_find_data.py
     deps:
-      - stages/01_download.sh
+      - ./stages/01_find_data.py
     outs:
-      - download
-      - list
-  unzip: 
-    cmd: stages/02_unzip.sh
+      - list/target_links.txt
+  download:
+    cmd: bash stages/02_download_data.sh
     deps:
-      - stages/02_unzip.sh
-      - download
-      - list
+      - ./stages/02_download_data.sh
     outs:
-      - raw
-  build: 
-    cmd: stages/03_build.sh
+      - download/
+  process:
+    cmd: python stages/03_process_data.py
```

Comment:
Use `python3` here.

Reply:
Resolved in eed890e86f91d674afd577ec2eef9fc0b44f54ea